src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -1101,6 +1101,11 @@ public override void MessageHandler(object sender, BuildMessageEventArgs e)
         {
             if (showOnlyErrors || showOnlyWarnings) return;
 
+            if (e.BuildEventContext == null && e is AssemblyLoadBuildEventArgs)
+            {
+                return;
```

Comment:
Should we not log this in the text log, too? It's generally less helpful than binlogs but still has some place, and is (sadly) still popular.

Reply:
**tl;dr;:** I believe it's out of scope currently.

It logs with exception of assembly loading for custom loggers. Those do not have any `BuildContext` that `ParallelConsoleLogger` relies upon quite a lot. It would take some refactoring of `ParallelConsoleLogger`, which is definitely doable but feels beyond scope of this change. 
If it would be missed - it can definitely be added later on.
Thoughts?